src/routing.ts

```diff
@@ -16,6 +17,15 @@ export interface Routing {
 
 export type Parsers = Partial<Record<ContentType, RequestHandler[]>>;
 
+export const createWrongMethodHandler =
+  (allowedMethods: Array<Method | AuxMethod>): RequestHandler =>
+  ({ method }, _res, next) =>
+    next(
+      createHttpError(405, `${method} is not allowed`, {
+        headers: { Allowed: allowedMethods.join(", ").toUpperCase() },
```

Comment:
what if there is a custom ResultHandler that does not care about setting headers from error?